requirements/common.txt

```diff
@@ -150,3 +150,11 @@ ujson==1.33
 
 uritemplate==0.6
 zope.interface==4.1.2
+talon==1.2.10
+cchardet==1.0.0
+cssselect==0.9.2
+lxml==3.6.0
+numpy==1.11.0
+regex==2016.6.19
+scikit-learn==0.16.1
+scipy==0.17.1
```

Comment:
This error seems to be related to this https://github.com/scikit-learn/scikit-learn/issues/4164.